app/v1/gov/govparams_util.go

```diff
@@ -37,13 +37,11 @@ func GetProposalLevel(p Proposal) ProposalLevel {
 
 func GetProposalLevelByProposalKind(p ProposalKind) ProposalLevel {
 	switch p {
-	case ProposalTypeTxTaxUsage, ProposalTypePlainText:
+	case ProposalTypePlainText:
 		return ProposalLevelNormal
-	case ProposalTypeParameterChange, ProposalTypeAddToken:
+	case ProposalTypeParameterChange, ProposalTypeAddToken, ProposalTypeTxTaxUsage:
```

Comment:
`ProposalTypeAddToken --> ProposalTypeTokenAddition`
`ProposalTypeTxTaxUsage --> ProposalTypeCommunityTaxUsage`

and related docs...